ansible/inventory.yml

```diff
@@ -9,8 +9,8 @@
     - role: gsa.datagov-deploy-apache2
       apache2_enabled: "{{ datagov_in_service | default(true) }}"
       tags: apache
-    - {role: software/ckan/inventory, tags: ['deploy']}
     - {role: software/ckan/saml2, tags: ['saml2'], when: "inventory_ckan_saml2_enabled"}
```

Comment:
I think this is going to break inventory.data.gov because the saml2 who.ini config is going to be overwritten with etc_ckan_who.default.ini.j2.

Reply:
Why do we want to drop saml2 role? Without the role, catalog and inventory are going to have same saml2 task duplicated?

Reply:
Catalog does not use the saml2 role, only Inventory. You've already moved the who.ini and production.ini configuration into the inventory role, so we may as well move the remaining tasks from the saml2 role and remove it.

Background:

The saml2 role is sort of flawed because it needs to modify both production.ini and who.ini, which are owned by the ckan role. That's why saml2 happens _after_ the ckan role, because it needs to be the last to modify production.ini and who.ini, otherwise the saml2 configuration is overwritten.

When we were working on the saml2 and native login options for catalog, it was easier to fold the saml2 pieces into the ckan role without impacting Inventory.

The ckan role should be shared between catalog and inventory, but it's a chunk of tech debt that we haven't tackled yet. Unfortunately, this isn't the first time we're copy/pasting tasks from the catalog role to inventory.